.changeset/silly-colts-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+"@biomejs/biome": patch
```

Comment:
Since this change primarily affects the wasm packages, probably should also include the wasm packages in this list.

Reply:
@dyc3 Actually we only use `@biomejs/biome` and `@biomejs/js-api` for changing what packages to ship. Bumping `@biomejs/biome` also ships WASM packages.

Reply:
Right, but when the changelogs get rendered out, the change only shows up for a package if its listed in the changeset. Each package has it's own changelog.

Reply:
I agree, WASM API changes should be listed in the changelogs for `@biome/wasm-*`. However, we don't render changelogs for packages other than `@biome/biome` and `@biome/js-api` for now, so adding WASM packages is meaningless. Let me align it with #6896 for this time.

Reply:
I think @dyc3 makes a good point. We usually publish everything together, but we can add a changelog line for those packages. I can update the changeset of the other PR